docs/getting-started.md

```diff
@@ -16,18 +16,26 @@ First, here are the things you need before starting:
   on the EV3. **All cards larger than 32GB will not work with the EV3!**
 * A computer with an adapter for the SD card. You will need administrator user
   permissions on this computer.
-* A way to communicate. For the EV3, this can be one of the following:
-    * USB cable (the one that comes with the EV3)
-    * USB WiFi dongle
-    * USB Ethernet (wired) dongle
-    * Bluetooth
+* A way to communicate.
+
+  For the EV3, this can be one of the following:
+
+  * USB cable (the one that comes with the EV3)
+  * USB WiFi dongle
+  * USB Ethernet (wired) dongle
+  * Bluetooth
+
   For Raspberry Pi you will need to use the wired Ethernet connection first.
+  With PiStorms the EV3 options will work too since you have a screen to configure
+  them on.
 
-<div class="alert alert-warning">
-    <span class="glyphicon glyphicon-alert"></span>
-    The instructions on this page only apply to releases dated September
-2015 or later. Older releases are no longer supported.
-</div>
+
+<span class="glyphicon glyphicon-alert"></span>
+The instructions on this page only apply to releases dated December
+2015 or later. Older releases are no longer supported. There are many major changes
+in this release, so if something seems broken or the documentation seems incorrect,
+please [open an issue on GitHub](../../support).
```

Comment:
This link should probably be relative to `site.github.url` to make sure we don't have broken links... not confident in relative links